pkg/operator/ceph/csi/predicate.go

```diff
@@ -41,6 +44,10 @@ func predicateController() predicate.Funcs {
 
 			// If a Ceph Cluster is created we want to reconcile the csi driver
 			if cephCluster, ok := e.Object.(*cephv1.CephCluster); ok {
+				// If there are more than one ceph cluster in the same namespace do not reconcile
+				if controller.DuplicateCephClusters(ctx, c, e.Object, false) {
```

Comment:
Do we really need to skip reconcile in the csi and obc controllers in this case? It's a rare enough error that it doesn't seem necessary to check here and just keep these other controllers simple.

Reply:
If we don't check, the logs will be a bit cluttered and users might skip the useful information. CSI and OBC controllers will reconcile and hide the message:

```
2021-12-17 09:27:02.826962 E | ceph-spec: found more than one ceph cluster in namespace "rook-ceph". not reconciling. only one ceph cluster per namespace.
2021-12-17 09:27:02.826989 E | ceph-spec: found ceph cluster "lesebb" in namespace "rook-ceph"
2021-12-17 09:27:02.826994 E | ceph-spec: found ceph cluster "rook-ceph" in namespace "rook-ceph"
```